src/appleseedmaya/exporters/meshexporter.cpp

```diff
@@ -435,32 +435,9 @@ MurmurHash MeshExporter::hash() const
     return m_hash;
 }
 
+// Insert mesh object params here.
 void MeshExporter::meshAttributesToParams(renderer::ParamArray& params)
 {
-    int mediumPriority = 0;
-    if (AttributeUtils::get(node(), "asMediumPriority", mediumPriority))
-        params.insert("medium_priority", mediumPriority);
```

Comment:
Did setting these parameters had any effect before this PR?

Reply:
Probably not.